examples/base_and_running_difference_mapcubes.py

```diff
@@ -0,0 +1,69 @@
+"""
+========================================
+Base Difference and Running Difference in Mapcubes
```

Comment:
`Base and Running Difference in Mapcubes`
Also make the `=` the same length as the title.

Reply:
Did CI broke due to this?

Reply:
Not sure, if you rebase or merge in master we should get a cleaner doc build log.